orca-clouddriver/src/main/groovy/com/netflix/spinnaker/orca/clouddriver/pipeline/manifest/DeleteManifestStage.java

```diff
@@ -33,6 +33,6 @@
   public void taskGraph(Stage stage, TaskNode.Builder builder) {
     builder.withTask(DeleteManifestTask.TASK_NAME, DeleteManifestTask.class)
         .withTask("monitorDelete", MonitorKatoTask.class)
-        .withTask(DeleteManifestForceCacheRefreshTask.TASK_NAME, DeleteManifestForceCacheRefreshTask.class);
```

Comment:
Looks like this is the only reference to that task - can it be deleted?

Reply:
Good catch